test/pools/LinearLiquidityPool/TestMulticoinTrading.sol

```diff
@@ -0,0 +1,245 @@
+pragma solidity >=0.6.0;
+
+import "truffle/Assert.sol";
+import "./BaseMulticon.sol";
+
+contract TestMulticoinTrading is Base {
+    event LogUint(string, uint);//Log event
+
+    StablecoinMock stablecoinA;//Stablecoin A
+    StablecoinMock stablecoinB;//Stablecoin B
+    StablecoinMock stablecoinC;//Stablecoin C
+
+    uint OptionsPrice;
+
+    function beforeEach() public{
+        stablecoinA=new StablecoinMock(18);
```

Comment:
There should be a spacing before and after the equeals sign: `stablecoinA = new StablecoinMock(18);`